rebasehelper/output_tools/text_output_tool.py

```diff
@@ -46,114 +47,132 @@ def print_success_message(cls):
     @classmethod
     def print_patches(cls, patches, summary):
         if not patches:
             logger_report.info("Patches were neither modified nor deleted.")
             return
-        logger_report.info(summary)
-        max_name = 0
-        for value in six.itervalues(patches):
-            if value:
-                new_max = max([len(os.path.basename(x)) for x in value])
-                if new_max > max_name:
-                    max_name = new_max
-        max_key = max([len(x) for x in six.iterkeys(patches)])
-        for key, value in six.iteritems(patches):
-            if value:
-                for patch in value:
-                    logger_report.info('Patch %s [%s]', os.path.basename(patch).ljust(max_name), key.ljust(max_key))
+
+        logger_report.info("Rebased patches are located in {}".format(cls.get_relative_results_path('rebased-sources')))
+        logger_report.info("Legend:")
+        logger_report.info("[-] = already applied, patch removed")
+        logger_report.info("[*] = merged, patch modified")
+        logger_report.info("[!] = conflicting or inapplicable, patch skipped")
+        logger_report.info("[ ] = patch untouched")
+
+        for patch_type, patch_list in sorted(six.iteritems(patches)):
+            if patch_list:
+                symbols = dict(deleted='-', modified='*', inapplicable='!')
+                for patch in sorted(patch_list):
+                    logger_report.info(' * {0:40} [{1}]'.format(os.path.basename(patch),
+                                                                symbols.get(patch_type, ' ')))
 
     @classmethod
-    def print_rpms(cls, rpms, version):
+    def print_rpms_and_logs(cls, rpms, version):
+        """
+        Prints information about location of RPMs and logs created during rebase
+        :param rpms: dictionary of (S)RPM paths
+        :param version: new/old version string
+        :return:
+        """
         pkgs = ['srpm', 'rpm']
         if not rpms.get('rpm', None):
             return
-        message = '\n{0} (S)RPM packages:'.format(version)
+        message = '\n{} packages'.format(version)
         cls.print_message_and_separator(message=message, separator='-')
         for type_rpm in pkgs:
             srpm = rpms.get(type_rpm, None)
             if not srpm:
                 continue
-            message = "%s package(s): are in directory %s :"
+
+            if type_rpm == 'srpm':
+                message = "\nSource packages and logs are in directory %s:"
+            else:
+                message = "\nBinary packages and logs are in directory %s:"
+
             if isinstance(srpm, str):
-                logger_report.info(message, type_rpm.upper(), os.path.dirname(srpm))
-                logger_report.info("- %s", os.path.basename(srpm))
+                # Print SRPM path
+                dirname = os.path.dirname(srpm)
+                logger_report.info(message, cls.get_relative_results_path(version.lower() + '-build/SRPM'))
```

Comment:
You should use `os.path.join()` here instead of `/` (or `os.path.sep`). But wouldn't it be better to make `get_relative_results_path()` method to accept multiple arguments which would be joined together?

Reply:
Also, you switched to using `format()` in logger methods, but here you are not using it. To be honest I'm not sure if we should get rid of lazy logging, it's true that it makes the code less consistent, but it isn't really the same thing as using the `%` operator.